docker/cloudbuild.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+# Build the main getsentry/sentry image
```

Comment:
You likely want to add a step here to first pull the `latest` tag so it can try and reuse layers if applicable instead of building from scratch every time. We do this elsewhere.

For example: https://github.com/getsentry/snuba/blob/master/cloudbuild.yaml#L2-L7

Reply:
Yup, will add those optimizations on a follow up.